components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -813,22 +813,52 @@ - (void)setHitAreaForButtons:(NSArray<MDCButton *> *)buttons {
   }
 }
 
-- (void)layoutHorizontalButtons:(NSArray<MDCButton *> *)buttons {
+- (void)layoutHorizontalButtons:(NSArray<MDCButton *> *)buttons actionSize:(CGSize)actionSize {
   UIEdgeInsets actionsInsets =
       self.enableAdjustableInsets ? self.actionsInsets : MDCDialogActionsInsets;
+  CGFloat horizontalMargin = (self.enableAdjustableInsets ? self.actionsHorizontalMargin
+                                                          : MDCDialogActionsHorizontalPadding);
+  CGFloat maxButtonWidth =
+      self.actionsScrollView.contentSize.width - (actionsInsets.left + actionsInsets.right);
   CGPoint buttonOrigin = CGPointZero;
-  buttonOrigin.x = self.actionsScrollView.contentSize.width - actionsInsets.right;
+  CGFloat buttonWidth = 0.f;
```

Comment:
I forget where we stand on the `.f` suffix for CGFloats... Is that a convention within our code?

Reply:
There was a decision that we should follow the official style guide (using .f), but no decision about changes to existing files...